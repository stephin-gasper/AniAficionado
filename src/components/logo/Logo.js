import React from 'react';
import {Image} from 'react-native';
import LogoIcon from '../../assets/images/logo.png';

const Logo = () => <Image style={{width: 80, height: 80}} source={LogoIcon} />;

export default Logo;
