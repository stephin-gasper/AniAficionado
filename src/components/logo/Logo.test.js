import React from 'react';
import {render} from 'test/test-utils';

import Logo from './Logo';

describe('<Logo />', () => {
  it('should render correctly', () => {
    expect(render(<Logo />)).toMatchSnapshot();
  });
});
