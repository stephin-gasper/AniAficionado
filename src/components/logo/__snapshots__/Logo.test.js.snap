// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Logo /> should render correctly 1`] = `
<Image
  source={
    Object {
      "testUri": "../../../src/assets/images/logo.png",
    }
  }
  style={
    Object {
      "height": 80,
      "width": 80,
    }
  }
/>
`;
